intro/fonts/index.html

```diff
@@ -78,8 +78,8 @@ <h2><a id="rendering-fonts" class="js"></a>Rendering fonts in Pygame</h1>
         <ul>
             <li><code>text</code>, which is a string with the text you want to render;</li>
             <li><code>antialias</code>, which is a boolean indicating whether you want your text to have smooth edges;</li>
-            <li><code>color</code>, which can be a <code>tuple[int, int, int]</code> or a string.</li>
-            <li><code>background</code>, which is a color value and can be a <code>tuple[int, int, int]</code> or a string.</li>
+            <li><code>color</code>, which is a <a href="/intro/color/">color value</a>;</li>
+            <li><code>background</code>, which is a <a href="/intro/color/">color value</a>.</li>
```

Comment:
`href` links link to a non-existent location currently and it seems they should link to `/intro/colors` anyways since that is what _Colors in pygame_ on homepage link to

Also, I think those direct type hints should be left in place otherwise readers would be required to go to another page

Reply:
It's fine that the linked page doesn't exist, because it hasn't been written yet. The reason that the type hints were removed was because they aren't the only possible options for color, but I think a couple tweaks can be made